src/lib/holocene/select/select.svelte

```diff
@@ -46,22 +48,28 @@
   {/if}
   <MenuContainer class="w-full">
     <MenuButton
-      class="select-input-container"
+      class="select-input-container {disabled ? 'disabled' : ''}"
```

Comment:
Best to do class:disabled when it's a boolean like this of either adding a class. Usually I prefer the inline when it's between two different class {disabled ? 'disabled' : 'enabled'}

Reply:
I did it this way so that the `disabled` class is only being passed down to the `MenuButton` for the `Select` component. 
The `DropdownButton` also uses the `MenuButton` but the `disabled` state (set for the `readOnly` option) does not have the `disabled` styling. Adding `class:disabled` results in 👇
| Before | After |
|---| ---|
|![Screen Shot 2022-08-09 at 9 24 22 AM](https://user-images.githubusercontent.com/15069288/183690524-d1d5d39b-ad8e-4d64-8756-9719050601d3.png)|![Screen Shot 2022-08-09 at 9 25 51 AM](https://user-images.githubusercontent.com/15069288/183690872-dd5df3ad-54f0-48c3-b93a-0c2dd30557bb.png)|
 
Maybe we could consider passing a `readOnly` prop to the `MenuButton`?

Reply:
Given the trade offs, that sounds reasonable.